lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -147,6 +147,7 @@ def render_to_fragment(self, request, program_uuid, **kwargs):  # lint-amnesty,
         }
         program_discussion_lti = ProgramDiscussionLTI(program_uuid, request)
         program_live_lti = ProgramLiveLTI(program_uuid, request)
+        discussion_or_live_configured = program_discussion_lti.is_configured or program_live_lti.is_configured
```

Comment:
We also need to show tabs if at least one pathway exists

Reply:
+1 for the pathways